release_docs/RELEASE.txt

```diff
@@ -47,6 +47,11 @@ New Features
 
     Configuration:
     -------------
+    - Deprecate and remove old SZIP library in favor of LIBAEC library
```

Comment:
Use past tense for consistency - `Deprecated`.

Reply:
Deprecated and removed ...